src/Silverpop.Client/TransactClient.cs

```diff
@@ -344,10 +348,16 @@ public static TransactClient Create(int podNumber, string username, string passw
         /// and appSettings from configuration. When a setting is set in both
         /// TransactClientConfigurationSection and appSettings the appSettings value is used.
         /// </summary>
+        /// <param name="configuration">
+        /// Optional provide IConfiguration to have associated app settings used.
+        /// </param>
         /// <param name="configProvider">
         /// Optional override for specifying a custom TransactClientConfigurationProvider.
         /// </param>
         public static TransactClient CreateUsingConfiguration(
+#if NETSTANDARD
+            IConfiguration configuration = null,
```

Comment:
Since this is limited to `NETSTANDARD`, what about not having a default-value and require someone to fix the breaking change. Otherwise, they might not understand why `configProvider.GetFromAppSettings` is throwing due to a `null` `IConfiguration` instance.

Reply:
All netstandard usages might not be able to construct (or, want to construct) an `IConfiguration`? Was trying to be careful with how it's implemented to not impact folks. Technically this is already a breaking change by not adding this at the end of the parameter list but I'm hoping the other parameter isn't used enough to matter.